spi/identity-hub-spi/src/main/java/org/eclipse/edc/identityhub/spi/IdentityHubApiContext.java

```diff
@@ -17,4 +17,5 @@
 public interface IdentityHubApiContext {
     String IDENTITY = "identity";
     String IH_DID = "did";
+    String PRESENTATION = "resolution"; // should be "presentation", but this would break config
```

Comment:
could we create an issue to deprecate "resolution" and start using "presentation"?

Reply:
#431